#Expyriment 1.0.0 (Python 3.10.5), .xpe-file, coding: UTF-8
#date: Sat Oct 11 2025 15:06:17
#sha1: 578b34
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Linux', node='eoghan-Precision-7530', release='6.8.0-85-generic', version='#85-Ubuntu SMP PREEMPT_DYNAMIC Thu Sep 18 15:26:59 UTC 2025', machine='x86_64')
Time,Type,Event,Value,Detail,Detail2
3295,design,log,
#design: #exp: Blindspot
#design: #dvn: block nb,trial nb,type,word,color,RT,accuracy
#design: block_cnt,block_id,trial_cnt,trial_id
3295,design,logged,
3295,Experiment,started
4115,Keyboard,received,13,wait
4128,Stimulus,presented,0
6044,Keyboard,received,121,wait
6935,Keyboard,received,121,wait
6937,Stimulus,presented,1
6943,stderr,received,'Traceback (most recent call last):'
6943,stderr,received,'  File "/home/eoghan/Documents/cog_sup/experimental_app/Assigment/Week-5/Exercises/stroop_dog.py", line 60, in <module>'
6943,stderr,received,'    run_block()'
6943,stderr,received,'  File "/home/eoghan/Documents/cog_sup/experimental_app/Assigment/Week-5/Exercises/stroop_dog.py", line 52, in run_block'
6943,stderr,received,'    run_trial(False)'
6943,stderr,received,'  File "/home/eoghan/Documents/cog_sup/experimental_app/Assigment/Week-5/Exercises/stroop_dog.py", line 37, in run_trial'
6943,stderr,received,'    text_display = colors[ind]'
6943,stderr,received,'IndexError: list index out of range'
